updating enterprise documentation with additional clarity (#7495)
hashicorp/consul: website/source/docs/enterprise/read-scale/index.html.md

page_title: "Consul Enterprise Enhanced Read Scalability"
sidebar_current: "docs-enterprise-read-scale"
description: |-
  Consul Enterprise supports increased read scalability without impacting write latency by introducing
  non-voting servers.
---

> # Enhanced Read Scalability / Non-Voting Servers

Reviewer: ```suggestion
# Enhanced Read Scalability with Non-Voting Servers
```

> [Consul Enterprise](https://www.hashicorp.com/consul.html) provides the ability to scale Consul server clusters

Reviewer: ```suggestion
[Consul Enterprise](https://www.hashicorp.com/consul.html) provides the ability to scale Consul servers
```

> to include voting and non-voting servers. Non-voting servers still receive data from the cluster replication

Reviewer: ```suggestion
to include voting and non-voting servers. Non-voting servers still receive data from the cluster replication,
```

> however do not take part in quorum election operations. Expanding your Consul cluster in this way can scale

Reviewer: ```suggestion
however, they do not take part in quorum election operations. Expanding your Consul cluster in this way can scale
```

reads without impacting write latency. 

> For more details, review the [Consul Server configuration](https://www.consul.io/docs/agent/options.html) 

Reviewer: ```suggestion
For more details, review the [Consul server configuration](https://www.consul.io/docs/agent/options.html) 
```
